client/components/credit-card/stored-card.jsx

```diff
@@ -0,0 +1,49 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+
+export const cardType = PropTypes.shape( {
+	card: PropTypes.string,
+	card_type: PropTypes.string,
```

Comment:
We'll probably want to clean up the 🐍 case here later

Reply:
Done in https://github.com/Automattic/wp-calypso/pull/24485/commits/646bea9c3b0278b73c80ce0cf34253645391eb02, as noted below. Thanks!